bot/seasons/christmas/adventofcode.py

```diff
@@ -196,14 +196,23 @@ def __init__(self, bot: commands.Bot):
     @adventofcode_group.command(name="join", aliases=("j",), brief="Learn how to join PyDis' private AoC leaderboard")
     async def join_leaderboard(self, ctx: commands.Context):
         """
-        Retrieve the link to join the PyDis AoC private leaderboard
+        DM the user the information for joining the PyDis AoC private leaderboard
         """
 
+        author = ctx.message.author
+        log.info(f"{author.name} ({author.id}) has requested the PyDis AoC leaderboard code")
+
         info_str = (
             "Head over to https://adventofcode.com/leaderboard/private "
             f"with code `{AocConfig.leaderboard_join_code}` to join the PyDis private leaderboard!"
         )
-        await ctx.send(info_str)
+        try:
+            await author.send(info_str)
+        except discord.errors.Forbidden:
+            log.debug(f"{author.name} ({author.id}) has disabled DMs from server members")
+            await ctx.send(
+                f':x: {author.mention}, please (temporarily) enable DMs to receive the join code'
```

Comment:
Why is the call split across 3 lines? It doesn't seem like it violates the line length limit.